### ISSUE TITLE
Error-Message after updating from php storm 2016.3 ->2016.3.1

### ISSUE THREAD

**Issue by ZYZ64738** [Haehnchen/idea-php-symfony2-plugin#830]
update failed for AnAction with ID=SymfonyWebDeploymentDownload: SERVER_CONFIG ...

[errorlog.txt](https://github.com/Haehnchen/idea-php-symfony2-plugin/files/643663/errorlog.txt)

**Comment by oleg-andreyev** [comment]
Having same issue:
PS 2016.3.1
```
update failed for AnAction with ID=SymfonyWebDeploymentDownload: SERVER_CONFIG
java.lang.NoSuchFieldError: SERVER_CONFIG
	at fr.adrienbrault.idea.symfony2plugin.webDeployment.actions.SymfonyWebDeploymentDownloadAction.update(SymfonyWebDeploymentDownloadAction.java:31)
	at com.intellij.openapi.actionSystem.ex.ActionUtil.performDumbAwareUpdate(ActionUtil.java:122)
	at com.intellij.openapi.actionSystem.impl.Utils.a(Utils.java:188)
	at com.intellij.openapi.actionSystem.impl.Utils.updateGroupChild(Utils.java:265)
	at com.intellij.openapi.actionSystem.impl.Utils.a(Utils.java:239)
	at com.intellij.openapi.actionSystem.impl.Utils.a(Utils.java:209)
	at com.intellij.openapi.actionSystem.impl.Utils.expandActionGroup(Utils.java:147)
	at com.intellij.openapi.actionSystem.impl.Utils.expandActionGroup(Utils.java:86)
	at com.intellij.openapi.actionSystem.impl.Utils.fillMenu(Utils.java:280)
	at com.intellij.openapi.actionSystem.impl.ActionMenu.a(ActionMenu.java:324)
	at com.intellij.openapi.actionSystem.impl.ActionMenu.access$600(ActionMenu.java:51)
	at com.intellij.openapi.actionSystem.impl.ActionMenu$MenuListenerImpl.menuSelected(ActionMenu.java:278)
	at javax.swing.JMenu.fireMenuSelected(JMenu.java:1037)
	at javax.swing.JMenu$MenuChangeListener.stateChanged(JMenu.java:1117)
	at javax.swing.DefaultButtonModel.fireStateChanged(DefaultButtonModel.java:349)
	at javax.swing.DefaultButtonModel.setSelected(DefaultButtonModel.java:232)
	at javax.swing.JMenu.setSelected(JMenu.java:305)
	at javax.swing.JMenu.menuSelectionChanged(JMenu.java:1186)
	at com.intellij.openapi.actionSystem.impl.ActionMenu.menuSelectionChanged(ActionMenu.java:240)
	at javax.swing.MenuSelectionManager.setSelectedPath(MenuSelectionManager.java:125)
	at javax.swing.plaf.basic.BasicMenuUI$Handler.mouseEntered(BasicMenuUI.java:526)
	at java.awt.Component.processMouseEvent(Component.java:6534)
	at javax.swing.JComponent.processMouseEvent(JComponent.java:3324)
	at java.awt.Component.processEvent(Component.java:6290)
	at java.awt.Container.processEvent(Container.java:2234)
	at java.awt.Component.dispatchEventImpl(Component.java:4881)
	at java.awt.Container.dispatchEventImpl(Container.java:2292)
	at java.awt.Component.dispatchEvent(Component.java:4703)
	at java.awt.LightweightDispatcher.retargetMouseEvent(Container.java:4898)
	at java.awt.LightweightDispatcher.retargetMouseEnterExit(Container.java:4686)
	at java.awt.LightweightDispatcher.trackMouseEnterExit(Container.java:4665)
	at java.awt.LightweightDispatcher.processMouseEvent(Container.java:4514)
	at java.awt.LightweightDispatcher.dispatchEvent(Container.java:4462)
	at java.awt.Container.dispatchEventImpl(Container.java:2278)
	at java.awt.Window.dispatchEventImpl(Window.java:2750)
	at java.awt.Component.dispatchEvent(Component.java:4703)
	at java.awt.EventQueue.dispatchEventImpl(EventQueue.java:758)
	at java.awt.EventQueue.access$500(EventQueue.java:97)
	at java.awt.EventQueue$3.run(EventQueue.java:709)
	at java.awt.EventQueue$3.run(EventQueue.java:703)
	at java.security.AccessController.doPrivileged(Native Method)
	at java.security.ProtectionDomain$1.doIntersectionPrivilege(ProtectionDomain.java:75)
	at java.security.ProtectionDomain$1.doIntersectionPrivilege(ProtectionDomain.java:86)
	at java.awt.EventQueue$4.run(EventQueue.java:731)
	at java.awt.EventQueue$4.run(EventQueue.java:729)
	at java.security.AccessController.doPrivileged(Native Method)
	at java.security.ProtectionDomain$1.doIntersectionPrivilege(ProtectionDomain.java:75)
	at java.awt.EventQueue.dispatchEvent(EventQueue.java:728)
	at com.intellij.ide.IdeEventQueue.d(IdeEventQueue.java:843)
	at com.intellij.ide.IdeEventQueue._dispatchEvent(IdeEventQueue.java:675)
	at com.intellij.ide.IdeEventQueue.dispatchEvent(IdeEventQueue.java:391)
	at java.awt.EventDispatchThread.pumpOneEventForFilters(EventDispatchThread.java:201)
	at java.awt.EventDispatchThread.pumpEventsForFilter(EventDispatchThread.java:116)
	at java.awt.EventDispatchThread.pumpEventsForHierarchy(EventDispatchThread.java:105)
	at java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:101)
	at java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:93)
	at java.awt.EventDispatchThread.run(EventDispatchThread.java:82)


java.lang.NoSuchFieldError: SERVER_CONFIG
	at fr.adrienbrault.idea.symfony2plugin.webDeployment.actions.SymfonyWebDeploymentDownloadAction.update(SymfonyWebDeploymentDownloadAction.java:31)
	at com.intellij.openapi.actionSystem.ex.ActionUtil.performDumbAwareUpdate(ActionUtil.java:122)
	at com.intellij.openapi.actionSystem.impl.Utils.a(Utils.java:188)
	at com.intellij.openapi.actionSystem.impl.Utils.expandActionGroup(Utils.java:139)
	at com.intellij.openapi.actionSystem.impl.Utils.expandActionGroup(Utils.java:86)
	at com.intellij.openapi.actionSystem.impl.Utils.fillMenu(Utils.java:280)
	at com.intellij.openapi.actionSystem.impl.ActionMenu.a(ActionMenu.java:324)
	at com.intellij.openapi.actionSystem.impl.ActionMenu.access$600(ActionMenu.java:51)
	at com.intellij.openapi.actionSystem.impl.ActionMenu$MenuListenerImpl.menuSelected(ActionMenu.java:278)
	at javax.swing.JMenu.fireMenuSelected(JMenu.java:1037)
	at javax.swing.JMenu$MenuChangeListener.stateChanged(JMenu.java:1117)
	at javax.swing.DefaultButtonModel.fireStateChanged(DefaultButtonModel.java:349)
	at javax.swing.DefaultButtonModel.setSelected(DefaultButtonModel.java:232)
	at javax.swing.JMenu.setSelected(JMenu.java:305)
	at javax.swing.JMenu.menuSelectionChanged(JMenu.java:1186)
	at com.intellij.openapi.actionSystem.impl.ActionMenu.menuSelectionChanged(ActionMenu.java:240)
	at javax.swing.MenuSelectionManager.setSelectedPath(MenuSelectionManager.java:125)
	at javax.swing.plaf.basic.BasicMenuUI$Handler.mouseEntered(BasicMenuUI.java:511)
	at java.awt.Component.processMouseEvent(Component.java:6534)
	at javax.swing.JComponent.processMouseEvent(JComponent.java:3324)
	at java.awt.Component.processEvent(Component.java:6290)
	at java.awt.Container.processEvent(Container.java:2234)
	at java.awt.Component.dispatchEventImpl(Component.java:4881)
	at java.awt.Container.dispatchEventImpl(Container.java:2292)
	at java.awt.Component.dispatchEvent(Component.java:4703)
	at java.awt.LightweightDispatcher.retargetMouseEvent(Container.java:4898)
	at java.awt.LightweightDispatcher.retargetMouseEnterExit(Container.java:4686)
	at java.awt.LightweightDispatcher.trackMouseEnterExit(Container.java:4665)
	at java.awt.LightweightDispatcher.processMouseEvent(Container.java:4514)
	at java.awt.LightweightDispatcher.dispatchEvent(Container.java:4462)
	at java.awt.Container.dispatchEventImpl(Container.java:2278)
	at java.awt.Window.dispatchEventImpl(Window.java:2750)
	at java.awt.Component.dispatchEvent(Component.java:4703)
	at java.awt.EventQueue.dispatchEventImpl(EventQueue.java:758)
	at java.awt.EventQueue.access$500(EventQueue.java:97)
	at java.awt.EventQueue$3.run(EventQueue.java:709)
	at java.awt.EventQueue$3.run(EventQueue.java:703)
	at java.security.AccessController.doPrivileged(Native Method)
	at java.security.ProtectionDomain$1.doIntersectionPrivilege(ProtectionDomain.java:75)
	at java.security.ProtectionDomain$1.doIntersectionPrivilege(ProtectionDomain.java:86)
	at java.awt.EventQueue$4.run(EventQueue.java:731)
	at java.awt.EventQueue$4.run(EventQueue.java:729)
	at java.security.AccessController.doPrivileged(Native Method)
	at java.security.ProtectionDomain$1.doIntersectionPrivilege(ProtectionDomain.java:75)
	at java.awt.EventQueue.dispatchEvent(EventQueue.java:728)
	at com.intellij.ide.IdeEventQueue.d(IdeEventQueue.java:843)
	at com.intellij.ide.IdeEventQueue._dispatchEvent(IdeEventQueue.java:675)
	at com.intellij.ide.IdeEventQueue.dispatchEvent(IdeEventQueue.java:391)
	at java.awt.EventDispatchThread.pumpOneEventForFilters(EventDispatchThread.java:201)
	at java.awt.EventDispatchThread.pumpEventsForFilter(EventDispatchThread.java:116)
	at java.awt.EventDispatchThread.pumpEventsForHierarchy(EventDispatchThread.java:105)
	at java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:101)
	at java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:93)
	at java.awt.EventDispatchThread.run(EventDispatchThread.java:82)
```

**Comment by Spurlos** [comment]
Thumbs up the first post if you have the same issue please

**Comment by thim81** [comment]
I had the same issue but I followed the tip from the readme.

> Plugin needs a valid PhpStorm indexer, use "File > Invalidate Caches / Restart" if something crazy is going on

This solved the issue for me.

**Comment by ZYZ64738** [comment]
Clearing PS-Cache like thim81s suggestion didn't solved it in my case. aftere clearing cache and restartinf PS i got imeediatly  the following error-message:
`update failed for AnAction with ID=SymfonyWebDeploymentDownload: SERVER_CONFIG
java.lang.NoSuchFieldError: SERVER_CONFIG
	at fr.adrienbrault.idea.symfony2plugin.webDeployment.actions.SymfonyWebDeploymentDownloadAction.update(SymfonyWebDeploymentDownloadAction.java:31)
	at com.intellij.openapi.actionSystem.ex.ActionUtil.performDumbAwareUpdate(ActionUtil.java:122)
	at com.intellij.openapi.actionSystem.impl.Utils.a(Utils.java:188)
	at com.intellij.openapi.actionSystem.impl.Utils.updateGroupChild(Utils.java:265)
	at com.intellij.openapi.actionSystem.impl.Utils.a(Utils.java:239)
	at com.intellij.openapi.actionSystem.impl.Utils.a(Utils.java:209)
	at com.intellij.openapi.actionSystem.impl.Utils.expandActionGroup(Utils.java:147)
	at com.intellij.openapi.actionSystem.impl.Utils.expandActionGroup(Utils.java:86)
	at com.intellij.openapi.actionSystem.impl.Utils.fillMenu(Utils.java:280)
	at com.intellij.openapi.actionSystem.impl.ActionMenu.a(ActionMenu.java:324)
	at com.intellij.openapi.actionSystem.impl.ActionMenu.access$600(ActionMenu.java:51)
	at com.intellij.openapi.actionSystem.impl.ActionMenu$MenuListenerImpl.menuSelected(ActionMenu.java:278)
	at javax.swing.JMenu.fireMenuSelected(JMenu.java:1037)
	at javax.swing.JMenu$MenuChangeListener.stateChanged(JMenu.java:1117)
	at javax.swing.DefaultButtonModel.fireStateChanged(DefaultButtonModel.java:349)
	at javax.swing.DefaultButtonModel.setSelected(DefaultButtonModel.java:232)
	at javax.swing.JMenu.setSelected(JMenu.java:305)
	at javax.swing.JMenu.menuSelectionChanged(JMenu.java:1186)
	at com.intellij.openapi.actionSystem.impl.ActionMenu.menuSelectionChanged(ActionMenu.java:240)
	at javax.swing.MenuSelectionManager.setSelectedPath(MenuSelectionManager.java:125)
	at javax.swing.plaf.basic.BasicMenuUI$Handler.mousePressed(BasicMenuUI.java:450)
	at java.awt.Component.processMouseEvent(Component.java:6522)
	at javax.swing.JComponent.processMouseEvent(JComponent.java:3324)
	at java.awt.Component.processEvent(Component.java:6290)
	at java.awt.Container.processEvent(Container.java:2234)
	at java.awt.Component.dispatchEventImpl(Component.java:4881)
	at java.awt.Container.dispatchEventImpl(Container.java:2292)
	at java.awt.Component.dispatchEvent(Component.java:4703)
	at java.awt.LightweightDispatcher.retargetMouseEvent(Container.java:4898)
	at java.awt.LightweightDispatcher.processMouseEvent(Container.java:4530)
	at java.awt.LightweightDispatcher.dispatchEvent(Container.java:4462)
	at java.awt.Container.dispatchEventImpl(Container.java:2278)
	at java.awt.Window.dispatchEventImpl(Window.java:2750)
	at java.awt.Component.dispatchEvent(Component.java:4703)
	at java.awt.EventQueue.dispatchEventImpl(EventQueue.java:758)
	at java.awt.EventQueue.access$500(EventQueue.java:97)
	at java.awt.EventQueue$3.run(EventQueue.java:709)
	at java.awt.EventQueue$3.run(EventQueue.java:703)
	at java.security.AccessController.doPrivileged(Native Method)
	at java.security.ProtectionDomain$1.doIntersectionPrivilege(ProtectionDomain.java:75)
	at java.security.ProtectionDomain$1.doIntersectionPrivilege(ProtectionDomain.java:86)
	at java.awt.EventQueue$4.run(EventQueue.java:731)
	at java.awt.EventQueue$4.run(EventQueue.java:729)
	at java.security.AccessController.doPrivileged(Native Method)
	at java.security.ProtectionDomain$1.doIntersectionPrivilege(ProtectionDomain.java:75)
	at java.awt.EventQueue.dispatchEvent(EventQueue.java:728)
	at com.intellij.ide.IdeEventQueue.d(IdeEventQueue.java:843)
	at com.intellij.ide.IdeEventQueue._dispatchEvent(IdeEventQueue.java:675)
	at com.intellij.ide.IdeEventQueue.dispatchEvent(IdeEventQueue.java:391)
	at java.awt.EventDispatchThread.pumpOneEventForFilters(EventDispatchThread.java:201)
	at java.awt.EventDispatchThread.pumpEventsForFilter(EventDispatchThread.java:116)
	at java.awt.EventDispatchThread.pumpEventsForHierarchy(EventDispatchThread.java:105)
	at java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:101)
	at java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:93)
	at java.awt.EventDispatchThread.run(EventDispatchThread.java:82)


java.lang.NoSuchFieldError: SERVER_CONFIG
	at fr.adrienbrault.idea.symfony2plugin.webDeployment.actions.SymfonyWebDeploymentDownloadAction.update(SymfonyWebDeploymentDownloadAction.java:31)
	at com.intellij.openapi.actionSystem.ex.ActionUtil.performDumbAwareUpdate(ActionUtil.java:122)
	at com.intellij.openapi.actionSystem.impl.Utils.a(Utils.java:188)
	at com.intellij.openapi.actionSystem.impl.Utils.expandActionGroup(Utils.java:139)
	at com.intellij.openapi.actionSystem.impl.Utils.expandActionGroup(Utils.java:86)
	at com.intellij.openapi.actionSystem.impl.Utils.fillMenu(Utils.java:280)
	at com.intellij.openapi.actionSystem.impl.ActionMenu.a(ActionMenu.java:324)
	at com.intellij.openapi.actionSystem.impl.ActionMenu.access$600(ActionMenu.java:51)
	at com.intellij.openapi.actionSystem.impl.ActionMenu$MenuListenerImpl.menuSelected(ActionMenu.java:278)
	at javax.swing.JMenu.fireMenuSelected(JMenu.java:1037)
	at javax.swing.JMenu$MenuChangeListener.stateChanged(JMenu.java:1117)
	at javax.swing.DefaultButtonModel.fireStateChanged(DefaultButtonModel.java:349)
	at javax.swing.DefaultButtonModel.setSelected(DefaultButtonModel.java:232)
	at javax.swing.JMenu.setSelected(JMenu.java:305)
	at javax.swing.JMenu.menuSelectionChanged(JMenu.java:1186)
	at com.intellij.openapi.actionSystem.impl.ActionMenu.menuSelectionChanged(ActionMenu.java:240)
	at javax.swing.MenuSelectionManager.setSelectedPath(MenuSelectionManager.java:125)
	at javax.swing.plaf.basic.BasicMenuUI$Handler.mouseEntered(BasicMenuUI.java:511)
	at java.awt.Component.processMouseEvent(Component.java:6534)
	at javax.swing.JComponent.processMouseEvent(JComponent.java:3324)
	at java.awt.Component.processEvent(Component.java:6290)
	at java.awt.Container.processEvent(Container.java:2234)
	at java.awt.Component.dispatchEventImpl(Component.java:4881)
	at java.awt.Container.dispatchEventImpl(Container.java:2292)
	at java.awt.Component.dispatchEvent(Component.java:4703)
	at java.awt.LightweightDispatcher.retargetMouseEvent(Container.java:4898)
	at java.awt.LightweightDispatcher.retargetMouseEnterExit(Container.java:4686)
	at java.awt.LightweightDispatcher.trackMouseEnterExit(Container.java:4665)
	at java.awt.LightweightDispatcher.processMouseEvent(Container.java:4514)
	at java.awt.LightweightDispatcher.dispatchEvent(Container.java:4462)
	at java.awt.Container.dispatchEventImpl(Container.java:2278)
	at java.awt.Window.dispatchEventImpl(Window.java:2750)
	at java.awt.Component.dispatchEvent(Component.java:4703)
	at java.awt.EventQueue.dispatchEventImpl(EventQueue.java:758)
	at java.awt.EventQueue.access$500(EventQueue.java:97)
	at java.awt.EventQueue$3.run(EventQueue.java:709)
	at java.awt.EventQueue$3.run(EventQueue.java:703)
	at java.security.AccessController.doPrivileged(Native Method)
	at java.security.ProtectionDomain$1.doIntersectionPrivilege(ProtectionDomain.java:75)
	at java.security.ProtectionDomain$1.doIntersectionPrivilege(ProtectionDomain.java:86)
	at java.awt.EventQueue$4.run(EventQueue.java:731)
	at java.awt.EventQueue$4.run(EventQueue.java:729)
	at java.security.AccessController.doPrivileged(Native Method)
	at java.security.ProtectionDomain$1.doIntersectionPrivilege(ProtectionDomain.java:75)
	at java.awt.EventQueue.dispatchEvent(EventQueue.java:728)
	at com.intellij.ide.IdeEventQueue.d(IdeEventQueue.java:843)
	at com.intellij.ide.IdeEventQueue._dispatchEvent(IdeEventQueue.java:675)
	at com.intellij.ide.IdeEventQueue.dispatchEvent(IdeEventQueue.java:391)
	at java.awt.EventDispatchThread.pumpOneEventForFilters(EventDispatchThread.java:201)
	at java.awt.EventDispatchThread.pumpEventsForFilter(EventDispatchThread.java:116)
	at java.awt.EventDispatchThread.pumpEventsForHierarchy(EventDispatchThread.java:105)
	at java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:101)
	at java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:93)
	at java.awt.EventDispatchThread.run(EventDispatchThread.java:82)
`

**Comment by BlackWiCKED** [comment]
Hey guys, there is a feature on GitHub to attach files instead of pasting the whole error log directly into the conversation, please use it. The person whom it's relevant will open it, while it won't annoy the rest.
The thread already looks like cr*p, do we really need to scroll down a mile to see the 4th comment? Ridiculous. If you can't add anything that will actually push the case and the conversation forward, then just be quiet and wait for the developer's response. Believe me, that's the faster way!
To summarize, looks like we all have the same issue that appeared after the last IDE update, and unfortunately clearing the cache only solves the problem temporarily... we're still waiting for a real solution.

**Comment by BlackWiCKED** [comment]
Take a look at #833, seems to be a duplicate thread but they identified the problem there.

**Comment by Haehnchen** [comment]
see #826

**Comment by axute** [comment]
same issue after update

**Comment by Haehnchen** [comment]
fixed by 0.13.133
